README.md

```diff
@@ -1,3 +1,5 @@
+NOTE (7/3/2024): This plugin is no longer being actively updated to newer versions of Coq. It works with Coq 8.9.
```

Comment:
Please clarify that the plugins that it uses, like PUMPKIN Pi and fix-to-elim and coq-plugin-lib, are still maintained, just not this top-level plugin.